### DISCUSSION TITLE
增加同步功能

### DISCUSSION THREAD

**Discussion by ajiangstudy** [Planshit/Tai#107] [Ideas]
能否增加使用OneDrive，坚果云或者github库进行同步，多台电脑希望实现无缝同步

**Comment by noberumotto** [comment]
这个有考虑过，但是短期内没办法。很多问题需要处理，我的能力有限，处理得很慢，有些历史遗留问题又得想办法推翻重写，所以现在工作量很大

**Reply by noberumotto** [reply]
请持续关注吧

**Reply by ajiangstudy** [reply]
好的，支持！
